### PR TITLE
Okteto files

### PR DESCRIPTION
**Pull request by paurosello**
Create files for live-editing the operator with okteto

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -142,6 +142,49 @@ used for tenant cluster nodes in `service/controller/key/ami.go`. The file is ge
 images have been published on AWS, this mapping can be updating using
 `devctl gen ami --dir service/controller/key`.
 
+## Live editing operator inside an installation
+
+- Download Okteto latest release from https://github.com/okteto/okteto/releases
+- `okteto init -n giantswarm`
+- Set correct label `app.giantswarm.io/branch: $BRANCH` in the manifest
+- Change your kubeconfig to the giantswarm namespace
+- Modify PSP of the current operator `kubectl patch psp aws-operator-$BRANCH-psp -p '{"spec":{"runAsGroup":{"ranges":null,"rule":"RunAsAny"},"runAsUser":{"rule":"RunAsAny"},"volumes":["secret","configMap","hostPath","persistentVolumeClaim","emptyDir"]}}'`
```
Comment (xh3b4sd) [inline README.md:151]: How can we undo these changes again?

**Review by xh3b4sd — APPROVED** [review]
🚀

File: README.md (hunk 1 above)
Comment (paurosello) [reply README.md:151]: Added command in last commit
